src/main/java/org/example/models/Band.java

```diff
@@ -0,0 +1,34 @@
+package org.example.models;
+
+public enum Band {
+
+    /**
+     * Enum of Band1 £20,000-£30,000 available for jobRoles.
+     */
+    BAND1("20,000-30,000"),
+    /**
+     * Enum of Band2 £30,001-£36,000 available for jobRoles.
+     */
+    BAND2("30,001-36,000"),
+    /**
+     * Enum of Band3 £36,001-£42,000 available for jobRoles.
+     */
+    BAND3("36,001-42,000"),
+    /**
+     * Enum of Band4 £42,001-£60,000 available for jobRoles.
+     */
+    BAND4("42,001-60,000");
```

Comment:
I don't think bands should link to salary, a band should be trainee, associate, senior associate, etc. Have a look at the useful links on the sprint board

Reply:
also making this an enum means we'll have to make a code change in the future if someone adds a new capability which we don't want to do. just use a string for band